django/tests/conftest.py

```diff
@@ -144,6 +144,7 @@ def mock_pdf_file2():
 
     with open(filename, "rb") as f:
         yield f
+    os.remove(filename)
```

Comment:
temp_file2.pdf wasn't removed after running tests locally, so you always had to manually delete it before commits

Reply:
Thanks!